examples/criterion/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "criterion_example"
+version = "0.1.0"
+authors = ["rhysd <https://rhysd.github.io>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+criterion = "0.3.0"
```

Comment:
Would you use `"*"` for version? Then I can notice when this action is incompatible with the latest Criterion version in the future.